source/common/runtime/runtime_features.cc

```diff
@@ -109,6 +109,7 @@ FALSE_RUNTIME_GUARD(envoy_reloadable_features_runtime_initialized);
 FALSE_RUNTIME_GUARD(envoy_reloadable_features_always_use_v6);
 // TODO(pradeepcrao) reset this to true after 2 releases (1.27)
 FALSE_RUNTIME_GUARD(envoy_reloadable_features_enable_include_histograms);
+FALSE_RUNTIME_GUARD(envoy_reloadable_features_upstream_http_filters_with_tcp_proxy);
```

Comment:
rock on - I was going to suggest this be false by default.  I assume y'all will test this internally and then flip it true?  If so can you please add a TODO?